src/JsonSchema/Constraints/BaseConstraint.php

```diff
@@ -89,6 +92,12 @@ public function addErrors(array $errors)
         }
     }
 
+    /**
+     * @param int $errorContext
```

Comment:
I would prefer the native type hint over a docblock

Reply:
added native typehints for some the stuff i already touched (and a bit more)